.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+sudo: false
+go:
+  - tip
+before_install:
+  - go get github.com/axw/gocov/gocov
+  - go get github.com/mattn/goveralls
+  - if ! go get github.com/golang/tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
```

Comment:
I got this line from goveralls doc, it seems [1] that after go1.5 cover is already in the standard repository, no need for this if, and we are not using gocov either, i'll remove them, thanks!

[1] https://godoc.org/golang.org/x/tools/cmd/cover